api/src/main/java/org/geysermc/geyser/api/GeyserApi.java

```diff
@@ -169,4 +170,13 @@ public interface GeyserApi extends GeyserApiBase {
     static GeyserApi api() {
         return Geyser.api(GeyserApi.class);
     }
+
+    /**
+     * Gets the {@link ApiVersion} representing the current Geyser api version
+     *
+     * @return the current geyser api version
+     */
+     default ApiVersion geyserApiVersion() {
+        return new ApiVersion(2, 4, 1);
```

Comment:
Should this not be more dynamic as hardcoding it like this feels like we might forget to change it?
Maybe add it to `gradle.properties` or use the existing version in there?

Reply:
https://github.com/GeyserMC/Geyser/pull/3880/commits/c6301b3b03eeceede94090e0726ed0eaec4ddffd added here - it'll parse the version in the gradle.properties file using blossom